packages/cinema/cinema.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import utils
+import tmdbsimple as tmdb
+import functools
+import random as ran
+import datetime as dt
+import json
+
+# Package database
+db = utils.db()['db']
+
+# Query
+Query = utils.db()['query']()
+
+
+def load_config(func):
+    @functools.wraps(func)
+    def wrapper_load_config(string, entities):
+        # Init "payload" as dictionary
+        payload = dict()
+        # Put data in payload
+        payload["string"] = string
+        payload["entities"] = entities
+        #  ISO 639-1 language code
+        payload["lang"] = utils.getqueryobj()["lang"][:2]
+        payload["today"] = dt.date.today()
+
+        # Load API key
+        payload["API_KEY"] = utils.config('API_KEY')
+        tmdb.API_KEY = payload["API_KEY"]
+        if payload["API_KEY"] == "YOUR_API_KEY":
+            return utils.output("end", "wrong_key", utils.translate("wrong_key"))
+
+        # Load words files who permit to check if user ask for a serie or a movie in every language
+        with open('packages/cinema/data/words/movie.json') as json_data:
+            payload["trl_movie"] = json.load(json_data)
+        with open('packages/cinema/data/words/serie.json') as json_data:
+            payload["trl_serie"] = json.load(json_data)
+
+        return func(payload)
+
+    return wrapper_load_config
+
+
+@load_config
+def recommend(payload):
```

Comment:
We should split this fiunctions in multiple small functions, the code is quite hard to understand if there weren't the comments, but it would be better to make it comprehensible without comments.